### ISSUE TITLE
Cursor in terminal disappears after tried to search by status incorrectly

### ISSUE THREAD

**Issue by anatoly-scherbakov** [ankitpokhrel/jira-cli#162]
**Describe the bug**

Illustrated on the recording:

![Peek 2021-10-15 16-21-cursor](https://user-images.githubusercontent.com/2282888/137464686-e34d2a34-c9c7-40e5-8dba-f819876517f4.gif)

**Please provide following details**
1. JiraCLI Version:
    ```
    (Version="0.0.0", GitCommit="6c1e78e4a3b57f50ffb3e2ab3c79faaf2117b9db", GoVersion="go1.17", BuildDate="2021-09-05"", Compiler="gc", Platform="linux/amd64")

    ```
2. Are you using Jira cloud or on-premise jira server?
    ```
    Jira cloud
    ```
3. What operating system are you using? Also mention version.
    ```
    Ubuntu 20.04.3 LTS
    ```
4. What terminal are you using? Also mention version.
    ```
    GNOME Terminal 3.36.2
    ```

**To Reproduce**

Please see the recording above.

**Expected behavior**

While the syntax I tried to use is clearly incorrect, it is not expected that the cursor disappears after this.

**Additional context**

Great tool!

**Comment by ankitpokhrel** [comment]
Thanks for opening the issue @anatoly-scherbakov, recording was quite helpful to understand the issue. 

I can reproduce the issue in my system too (macOS with iterm2 and zsh) and I think I know what's happening. I will possibly work on the fix this weekend.

**Comment by ankitpokhrel** [comment]
The issue should be fixed by #163. Please feel free to comment or open a new issue in case this is still not working for you.
